chunks/intro/components/Cover.js

```diff
@@ -49,6 +53,7 @@ export default class Intro extends Component {
       <div style={{ marginTop: '30px' }}>
         <Button
           raised
+          onClick = { () => this.goTo('/about') }
```

Comment:
I think you don't need to use an anonymus function
```suggestion
          onClick = { this.goTo('/about') }
```